test/create_test_environment.sh

```diff
@@ -58,7 +58,13 @@ fi
 # Display every command executed from now
 set -x
 
-gcloud projects create $GOOGLE_CLOUD_PROJECT
+gcloud projects create $GOOGLE_CLOUD_PROJECT || read -p "Looks like this service accounts exists. If so, continue? (y/n)? " CONT
```

Comment:
Does this command return an error when a service account already exists, or the project? I am not really keen to add complexity in this script and handle the various cases, it is meant as a simple helper to set up the test environment + documentation on how to do so manually if needed.